Nos/Views/CircularFollowButton.swift

```diff
@@ -0,0 +1,62 @@
+import CoreData
+import Dependencies
+import Logger
+import SwiftUI
+
+struct CircularFollowButton: View {
```

Comment:
I would like a doc comment here explaining the intended use of this button.

Reply:
Done!